misk/src/main/kotlin/misk/web/actions/WebActions.kt

```diff
@@ -2,18 +2,29 @@ package misk.web.actions
 
 import com.google.common.collect.Lists
 import misk.Chain
-import misk.Interceptor
+import misk.ApplicationInterceptor
+import misk.NetworkChain
+import misk.NetworkInterceptor
 import misk.web.RealChain
+import misk.web.RealNetworkChain
+import misk.web.Request
 import kotlin.reflect.KFunction
 import kotlin.reflect.KParameter
 
+fun WebAction.asNetworkChain(
```

Comment:
Is this used? In ``BoundActionMatch.handle`` we instantiate the ``RealNetworkChain`` directly

Reply:
Whoops, good catch. This is the second half of a refactor.